deployment/ccip/changeset/add_lane.go

```diff
@@ -15,25 +17,103 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/router"
 )
 
+var _ deployment.ChangeSet[AddLanesConfig] = AddLanesWithTestRouter
+
 type InitialPrices struct {
 	LinkPrice *big.Int // USD to the power of 18 (e18) per LINK
 	WethPrice *big.Int // USD to the power of 18 (e18) per WETH
 	GasPrice  *big.Int // uint224 packed gas price in USD (112 for exec // 112 for da)
 }
 
+type Lane struct {
+	From uint64
+	To   uint64
+}
+
+type AddLanesConfig struct {
+	FeeQuoterDestChainConfigArgs map[uint64]map[uint64]fee_quoter.FeeQuoterDestChainConfig
+	InitialPricesByChain         map[uint64]InitialPrices
+	ChainPairs                   []Lane
```

Comment:
```suggestion
	Lanes                   []Lane
```

Seems like a natural name for a slice of lane objects